apps/studio/components/interfaces/Integrations/Landing/AvailableIntegrations.tsx

```diff
@@ -22,16 +22,20 @@ export const AvailableIntegrations = () => {
   // available integrations for install
   const integrationsByCategory =
     selectedCategory === 'all'
-      ? availableIntegrations.filter((i) => !installedIds.includes(i.id))
-      : availableIntegrations.filter(
-          (i) => !installedIds.includes(i.id) && i.type === selectedCategory
-        )
+      ? availableIntegrations
+      : availableIntegrations.filter((i) => i.type === selectedCategory)
   const filteredIntegrations = (
     search.length > 0
       ? integrationsByCategory.filter((i) => i.name.toLowerCase().includes(search.toLowerCase()))
       : integrationsByCategory
   ).sort((a, b) => a.name.localeCompare(b.name))
 
+  const CATEGORIES = [
+    { key: 'all', label: 'All Integrations' },
+    { key: 'wrapper', label: 'Wrapper' },
+    { key: 'postgres_extension', label: 'Postgres Modules' },
```

Comment:
Singular `Wrapper` vs plural `Modules`? Which one is correct?

Reply:
def wrapperS - good catch